### DISCUSSION TITLE
Scraping ICML 2024 workshops

### DISCUSSION THREAD

**Discussion by oshapio** [george-gca/ai_papers_scrapper#2] [Q&A]
Thanks for a great resource! Are there any plans to add the ability to scrape ICML workshops, too? Otherwise, is there a recommended path to add this functionality? Thanks!

**Comment by george-gca** [comment]
I plan to do so, but I am still struggling with the openreview api. Probably someday in the near future I can make it work though.

**Reply by george-gca** [reply]
Implemented in https://github.com/george-gca/ai_papers_scrapper/commit/84efd8caefda251579b4d22791ac468d8d18d985
